firewood/src/merkle_util.rs

```diff
@@ -64,7 +64,7 @@ impl<S: ShaleStore<Node> + Send + Sync, T: BinarySerde> MerkleSetup<S, T> {
             .map_err(|_err| DataStoreError::GetError)
     }
 
-    pub const fn get_root(&self) -> DiskAddress {
+    pub const fn get_sentinel(&self) -> DiskAddress {
```

Comment:
question: Why is this pub?

Why not return the actual node here? I think all callers will always want to fetch that node right away anyway.

As always, feel free to defer that to another PR. I realize you only changed the name.

If you keep this name, I'd prefer `get_sentinel_address` or similar.